src/main/java/onlinebookstore/dto/BookDto.java

```diff
@@ -0,0 +1,21 @@
+package onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
+    private Long id;
+
+    private String title;
+
+    private String author;
+
+    private String isbn;
+
+    private BigDecimal price;
+
+    private String description;
+
+    private String coverImage;
```

Comment:
i think it should be like this:
```suggestion
    private String title;
    private String author;
    private String isbn;
    private BigDecimal price;
    private String description;
    private String coverImage;
```